src/services/cheWorkspaceClient/index.ts

```diff
@@ -26,13 +29,18 @@ export class CheWorkspaceClient {
   private websocketContext: string;
   private _restApiClient: IRemoteAPI;
   private _jsonRpcMasterApi: IWorkspaceMasterApi;
+  private _failingWebSockets: string[];
+  private webSocketEventEmitter: EventEmitter;
+  private webSocketEventName = 'websocketChanged';
```

Comment:
If I understand code correctly, I wonder if Che Workspace Client should expose available event names, instead of creating constant in each of client. But it does not block the current PR, but just small refactoring suggestion for later.